EfEnumToLookup/LookupGenerator/MetadataHandler.cs

```diff
@@ -53,15 +53,17 @@ private static IEnumerable<EnumReference> ProcessEdmProperties(IEnumerable<EdmPr
 			// get mapped table name from mapping, or fall-back to just the name if no mapping is set,
 			// I have no idea what causes Table to be null, and I have no unit test for it yet, but I have seen it.
 			var table = mappingFragment.StoreEntitySet.Table ?? mappingFragment.StoreEntitySet.Name;
-
-			foreach (var edmProperty in properties)
+            var schema = mappingFragment.StoreEntitySet.Schema ?? "dbo";

```

Comment:
defaulting to 'dbo' might cause problems with implementing non-sql-server variants (which have been requested). probably better to leave it unqualified if no schema present